hack/generate/samples/ansible/generate.go

```diff
@@ -43,24 +39,7 @@ var memcachedGVK = schema.GroupVersionKind{
 }
 
 func getCli() *cli.CLI {
-	ansibleBundle, _ := plugin.NewBundleWithOptions(
-		plugin.WithName(golang.DefaultNameQualifier),
-		plugin.WithVersion(ansible.Plugin{}.Version()),
-		plugin.WithPlugins(kustomizev2.Plugin{}, ansible.Plugin{}),
-	)
-
-	c, err := cli.New(
-		cli.WithCommandName("cli"),
-		cli.WithVersion("v0.0.0"),
-		cli.WithPlugins(
-			ansibleBundle,
-		),
-		cli.WithDefaultPlugins(cfgv3.Version, ansibleBundle),
-		cli.WithDefaultProjectVersion(cfgv3.Version),
-		cli.WithCompletion(),
-	)
-	pkg.CheckError("getting cli implementation:", err)
-	return c
+	return ansiblecli.GetPluginsCLI()
```

Comment:
We should not have this code here.
We must ensure that the hack generator uses the binary instead of mock the CLI and use it.
The binary must be generated locally and used to ensure that all works fine

c/c @acornett21 ^